site/docs/tutorial/ios-app.md

```diff
@@ -104,24 +104,36 @@ Enter the following at the command line:
 
 ```bash
 touch $WORKSPACE/WORKSPACE
+open -a Xcode $WORKSPACE/WORKSPACE
 ```
 
-This creates the empty `WORKSPACE` file.
+This creates and open the empty `WORKSPACE` file.
 
 ### Update the WORKSPACE file
 
 To build applications for Apple devices, Bazel needs to pull the latest
 [Apple build rules](https://github.com/bazelbuild/rules_apple) from its GitHub
-repository. To enable this, add the following to your `WORKSPACE` file:
+repository. To enable this, add the following 
+[`http_archive`](../be/workspace.html#http_archive) rule to your `WORKSPACE` 
+file:
 
 ```
-git_repository(
+http_archive(
     name = "build_bazel_rules_apple",
-    remote = "https://github.com/bazelbuild/rules_apple.git",
-    commit = "7ea0557",
+    strip_prefix = "rules_apple-0.1.0",
+    urls = ["https://github.com/bazelbuild/rules_apple/archive/0.1.0.tar.gz"],
 )
 ```
 
+**NOTE:** Please check the 
```

Comment:
Let's simplify this sentence:

"Always use the [latest version of the `build_apple` rules](https://github.com/bazelbuild/rules_apple/releases) in the `urls` and `strip_prefix` attributes."

Reply:
Fixed